receivers/pagerduty/pagerduty.go

```diff
@@ -1,11 +1,14 @@
 package pagerduty
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
 	"fmt"
 	"strings"
 
+	"github.com/alecthomas/units"
```

Comment:
I am kinda on the fence about bringing this dependency for the sake of humanized bytes sizes... I think I am leaning towards keeping it because the upstream alertmaanger uses it also.

Reply:
I went through the same thought process, ultimately deciding on keeping it for the same reason.